cmd/kubelet/app/options/options.go

```diff
@@ -199,6 +199,7 @@ func (s *KubeletServer) AddFlags(fs *pflag.FlagSet) {
 	fs.StringVar(&s.CgroupRoot, "cgroup-root", s.CgroupRoot, "Optional root cgroup to use for pods. This is handled by the container runtime on a best effort basis. Default: '', which means use the container runtime default.")
 	fs.StringVar(&s.ContainerRuntime, "container-runtime", s.ContainerRuntime, "The container runtime to use. Possible values: 'docker', 'rkt'. Default: 'docker'.")
 	fs.DurationVar(&s.RuntimeRequestTimeout.Duration, "runtime-request-timeout", s.RuntimeRequestTimeout.Duration, "Timeout of all runtime requests except long running request - pull, logs, exec and attach. When timeout exceeded, kubelet will cancel the request, throw out an error and retry later. Default: 2m0s")
+	fs.DurationVar(&s.ImagePullingStuckTimeout.Duration, "image-pull-stuck-timeout", s.ImagePullingStuckTimeout.Duration, "Stuck timeout of pulling image. If no pulling progress is made before this timeout, the image pulling will be cancelled. Default: 1m0s.")
```

Comment:
Is `stuck` the right word?  We usually use "progress" to denote this concept in Deployments.  I think I'd recommend `image-pull-progress-deadline` to be consistent with wording elsewhere in the API for deployments (deployment progress and deadline)